the-blue-alliance-ios/View Elements/Match/Breakdown/MatchBreakdownConfigurator2020.swift

```diff
@@ -0,0 +1,258 @@
+import Foundation
+import UIKit
+
+private class BreakdownStyle2020 {
+    public static let bottomImage = UIImage(systemName: "rectangle")
+    public static let outerImage = UIImage(systemName: "hexagon")
+    public static let innerImage = UIImage(systemName: "circle")
+    public static let checkImage = UIImage(systemName: "checkmark")
+    public static let xImage = UIImage(systemName: "xmark")
+}
+
+struct MatchBreakdownConfigurator2020: MatchBreakdownConfigurator {
+
+    static func configureDataSource(_ snapshot: inout NSDiffableDataSourceSnapshot<String?, BreakdownRow>, _ breakdown: [String: Any]?, _ red: [String: Any]?, _ blue: [String: Any]?) {
+        
+        guard let red = red, let blue = blue else {
```

Comment:
nit: Everything here takes an optional, we shouldn't *need* to unwrap like this.